images/istio/tests/main.tf

```diff
@@ -29,19 +33,126 @@ data "oci_exec_test" "operator-version" {
   script = "docker run --rm $IMAGE_NAME version"
 }
 
-data "oci_string" "ref" { input = var.digests.operator }
+data "oci_string" "operator-ref" { input = var.digests.operator }
+data "oci_string" "proxy-ref" { input = var.digests.proxy }
 
 resource "helm_release" "operator" {
-  name = "operator"
-
-  # There's no official helm chart for the istio operator
+  name             = "operator"
+  namespace        = local.namespace
+  create_namespace = true
+  # there's no official helm chart for the istio operator
   repository = "https://stevehipwell.github.io/helm-charts/"
   chart      = "istio-operator"
 
   values = [jsonencode({
     image = {
-      repository = data.oci_string.ref.registry_repo
-      tag        = data.oci_string.ref.pseudo_tag
+      repository = data.oci_string.operator-ref.registry_repo
+      tag        = data.oci_string.operator-ref.pseudo_tag
```

Comment:
Is it possible to point it at our images as well?

Reply:
This is our `operator` image. I think you mean pointing at our proxy and pilot image? That would be part of the IstioOperator resource that we will install. Will follow up with an additional test for that.